src/templates/__tests__/readme.test.js

```diff
@@ -84,10 +88,14 @@ describe("Templates - Readme", () => {
       expect(addDescription("")).toEqual("");
     });
   });
+
   describe("addBookwormsDescription", () => {
     test("should return description for file comment", () => {
       expect(addBookwormsDescription(1633027394454)).toEqual(
-        "_These bookmarks were last updated on 30/09/2021, 19:43:14 using [Bookworms](https://github.com/thearegee/bookworms)_"
+        expect.stringContaining("_These bookmarks were last updated on"),
```

Comment:
I assume this is because of potential timezones when the test is being run? Maybe we can mock the timer better to solve this issue?

Reply:
Not only timezone but also the format of the datetime comes from users locale. We could use some specific locale and timezone but maybe mocking is better option